src/observer.ts

```diff
@@ -1,35 +1,25 @@
 import { Subscription } from './subscription';
 
-/** Symbol.observable addition */
-/* Note: This will add Symbol.observable globally for all TypeScript users,
-  however, we are no longer polyfilling Symbol.observable */
+// Older versions of Rx will polyfill Symbol.observable, which gets
+// compiled into our UMD bundle. At runtime, our UMD bundle defines its
+// version of Symbol.observable, and since it's not getting it from Rx via
+// `require()` anymore, Rx defines and looks for a different Symbol.observable,
+// instance, leading to mismatches.
+// Assigning the global Symbol.observable to the one we bundle in here means
+// that Rx's polyfill will pick it up. Alternatively if there's already a global
+// Symbol.observable (like if Rx was required first), we should use that one inside Ix.
+
+// Symbol.observable addition
+// Note: This will add Symbol.observable globally for all TypeScript users
 declare global {
   interface SymbolConstructor {
     readonly observable: symbol;
   }
 }
 
-/** Symbol.observable or a string "@@observable". Used for interop */
-//tslint:disable-next-line
-const { observable: Symbol_observable } = require('rxjs/symbol/observable');
-
-let symbolObservable = Symbol_observable;
-if (typeof Symbol !== 'undefined') {
-  // Older versions of Rx will polyfill Symbol.observable, which gets
-  // compiled into our UMD bundle. At runtime, our UMD bundle defines its
-  // version of Symbol.observable, and since it's not getting it from Rx via
-  // `require()` anymore, Rx defines and looks for a different Symbol.observable,
-  // instance, leading to mismatches.
-  // Assigning the global Symbol.observable to the one we bundle in here means
-  // that Rx's polyfill will pick it up. Alternatively if there's already a global
-  // Symbol.observable (like if Rx was required first), we should use that one inside Ix.
-  if (typeof Symbol['observable'] === 'undefined') {
-    (Symbol as any)['observable'] = symbolObservable;
-  } else {
-    symbolObservable = Symbol['observable'];
-  }
-}
-export { symbolObservable };
+// Symbol.observable or a string "@@observable". Used for interop.
+// Referenced via string indexer so closure-compiler doesn't mangle.
+export const observable = (typeof Symbol === 'function' && Symbol.observable) || '@@observable';
```

Comment:
`observable` seems like a weird name in the context of this file, I would name this `symbolObservable`. Or put it into its own file `symbol.ts`.